query-engine/connector-test-kit/src/test/scala/util/Project.scala

```diff
@@ -19,9 +19,14 @@ case class Project(
   val dataSourceConfig: String = {
     val config = ConnectorConfig.instance
 
+    val provider = config.provider.stripSuffix("56") match {
+      case "vitess" => "mysql"
+      case provider => provider
+    }
+
     s"""
            |datasource test {
-           |  provider = "${config.provider.stripSuffix("56")}"
```

Comment:
Huh why does this stripping exist?

Reply:
It's legacy stuff. Don't really need to know...